cpp/src/transform/jit/kernel.cu

```diff
@@ -86,7 +99,7 @@ struct scalar {
   static constexpr int32_t index = Accessor::index;
 
   static __device__ decltype(auto) element(cudf::mutable_column_device_view_core const* outputs,
-                                           cudf::size_type row)
+                                           [[maybe_unused]] cudf::size_type row)
```

Comment:
If the parameter is unused, do not mention it instead.
```suggestion
                                           cudf::size_type)
```

Reply:
ok